### PR TITLE
Adds single site removal for geo-repl

### PR DESCRIPTION
**Pull request by stevsmit**
Standalone: https://stevsmit.github.io/quay-docs/single-site-geo/#standalone-georepl-site-removal

Operator: https://stevsmit.github.io/quay-docs/single-site-geo/#operator-georepl-site-removal

### CODE REVIEW

File: modules/operator-georepl-site-removal.adoc [hunk 1]
```diff
@@ -0,0 +1,63 @@
...
+[source,terminal]
+----
+sh-4.4$ python -m util.removelocation euwest2
+----
++
+.Example output
++
+[source,terminal]
+----
+/app/lib/python3.9/site-packages/tzlocal/unix.py:141: SyntaxWarning: "is not" with a literal. Did you mean "!="?
```
Comment (LiZhang19817) [inline modules/operator-georepl-site-removal.adoc:53]: These is the issue of https://issues.redhat.com/browse/PROJQUAY-5650, and will be fixed in Quay 3.9.0

Comment (stevsmit) [reply modules/operator-georepl-site-removal.adoc]: Removed. Thanks.

File: modules/operator-georepl-site-removal.adoc [hunk 2]
```diff
@@ -0,0 +1,59 @@
...
+[id="operator-georepl-site-removal"]
+= Removing a geo-replicated site from your {productname} Operator deployment
+
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You are logged into {ocp}.
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags.
```
Comment (LiZhang19817) [inline modules/operator-georepl-site-removal.adoc:11]: This description is not correct, actually only the imagestorageplacement is associated with storagelocation.

File: modules/standalone-georepl-site-removal.adoc [hunk 3]
```diff
@@ -0,0 +1,74 @@
+:_content-type: PROCEDURE
+[id="standalone-georepl-site-removal"]
+= Removing a geo-replicated site from your standalone {productname} deployment
+
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags.
```
Comment (LiZhang19817) [inline modules/standalone-georepl-site-removal.adoc:10]: This description is not correct, actually only the imagestorageplacement is associated with storagelocation.

File: modules/standalone-georepl-site-removal.adoc [hunk 4]
```diff
@@ -0,0 +1,74 @@
...
+By using the following procedure, {productname} administrators can remove sites in a geo-replicated setup. 
+
+.Prerequisites 
+
+* You have configured {productname} geo-replication with at least two sites, for example, `usstorage` and `eustorage`. 
+* Each site has its own Organization, Repository, and image tags. 
+
+.Procedure 
+
+. Enter the following command to execute a shell inside of the database pod:
```
Comment (LiZhang19817) [inline modules/standalone-georepl-site-removal.adoc:14]: It's not correct, the command to remove site should be executed inside Quay APP POD, not quay database pod

File: modules/operator-georepl-site-removal.adoc [hunk 5]
```diff
@@ -0,0 +1,59 @@
...
+* Each site has its own Organization, Repository, and image tags. 
+
+.Procedure 
+
+. Enter the following command to identify your `Quay` database pods:
++
+[source,terminal]
+----
+$ oc get pod -l quay-component=postgres -n  <quay-namespace> -o jsonpath='{.items[0].metadata.name}'
+----
```
Comment (LiZhang19817) [inline modules/operator-georepl-site-removal.adoc:20]: it's not necessary to get the Quay Database POD, customers only need to rsh to the Quay App POD to execute the command to remove site

File: modules/standalone-georepl-site-removal.adoc [hunk 6]
```diff
@@ -0,0 +1,74 @@
...
++
+[source,terminal]
+----
+bash-4.4$ python -m util.removelocation
+----
++
+.Example output
++
+[source,terminal]
+----
```
Comment (LiZhang19817) [inline modules/standalone-georepl-site-removal.adoc:67]: When Quay Customers execute the command to remove site, although they will see the following error message, the site can be removed successfully.
https://issues.redhat.com/browse/PROJQUAY-5650

File: modules/standalone-georepl-site-removal.adoc (hunk 3 above)
Comment (ibazulic) [reply modules/standalone-georepl-site-removal.adoc:10]: @LiZhang19817 there are three tables that govern blob storage in Quay:

1) `imagestorage`: contains references to all blobs stored in the registry.
2) `imagestoragelocation`: contains the names of all defined storage engines.
3) `imagestorageplacement`: a relational table that connects `imagestorage` with `imagestoragelocation`, i.e. every blob stored in the registry with a particular defined storage engine.

I'm not sure what you mean with your comment and how is it relevant to what you quoted here. Can you please explain what you meant?

Comment (LiZhang19817) [reply modules/standalone-georepl-site-removal.adoc:10]: @ibazulic Quay Geo-Replication means multiple Quay deployments across different sites, each organization created not belonging to any site, why here says **"Each site has its own Organization"**, for example when login from Quay site1 with users 'user1', it's not reasonable to say site1 own this new org.

Comment (ibazulic) [reply modules/standalone-georepl-site-removal.adoc:10]: @LiZhang19817 I know what georeplication means and how it works :-) Yep, now that I read it again, I understand better what you meant. Thanks for the additional explanation!

**Pull request merged**